### DISCUSSION TITLE
Error in Security onion using the command "so-soc-restart"

### DISCUSSION THREAD

**Discussion by nooraSalem** [Security-Onion-Solutions/securityonion#11692] [Unsupported Versions]
Hello Guys,

I have a security onion with the version 2.3.270 and when I write the command "so-soc-restart" I get an error that say that their is no soc available as shown in the screenshot.
<img width="413" alt="Screenshot 2023-11-02 191249" src="https://github.com/Security-Onion-Solutions/securityonion/assets/149691787/daa94453-7130-4c12-b567-cc66dbeac972">

**Comment by dougburks** [comment]
After you run the command, does SOC restart correctly?

Have you modified any configuration?

If you need further help, please provide more details about your system as shown at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720.
